test/custom_class/t/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+cmake_minimum_required(VERSION 3.1 FATAL_ERROR)
```

Comment:
It would be good to get build coverage in CI on this stuff (if there isn't already, i'm not sure) so we can make sure the API doesn't break on windows or something random. Maybe we can add this directory as a dependency from test/cpp/jit? @suo thoughts?

Reply:
I think, for now, while it's in such large flux, I'm not gonna bother with adding tests to CI and such.